scio-test/src/main/scala/com/spotify/scio/testing/RunEnforcementJobTest.scala

```diff
@@ -48,6 +49,12 @@ trait RunEnforcementJobTest extends AnyFlatSpec { this: PipelineSpec =>
     jt
   }
 
+  def jobTest(job: ScioContext => Any): InnerJobTest.Builder = {
```

Comment:
@clairemcginty @kellen makbe this should be kept capitalized, as the version with the type. WDYT ?

```suggestion
  def JobTest(job: ScioContext => Any): InnerJobTest.Builder = {
```